### ISSUE TITLE
Accessibility:  ScreenReader does not convey information about the required fields

### ISSUE THREAD

**Issue by jithinjosepkl** [wixtoolset/issues#5996]
* Which version of WiX are you building with?

3.10.3.3010

* Which version of Visual Studio are you building with (if any)?

VS 2017

* Which version of .NET are you building with?

4.0

* If the problem occurs when installing your packages built with WiX, what is the version of Windows the package is running on?

Windows 10

* Describe the problem and the steps to reproduce it.

ScreenReader does not convey information about the required fields on EndUser agreement popup. Please see attached video for more details.
[3759826_A11y_MSMPI_EndUserLicenseAgreementexe_ScreenReader.zip](https://github.com/wixtoolset/issues/files/3139917/3759826_A11y_MSMPI_EndUserLicenseAgreementexe_ScreenReader.zip)

* Describe the behavior you expected and how it differed from the actual behavior.

ScreenReader should announce about Required fields present on this popup.

## Feature requests

If this issue is a feature request:

* Describe the scenario and benefits that the feature supports.

> (Provide lots of specific detail so we understand the context around the feature.)

* Describe how you're accomplishing the feature today (if possible).

> (Be as specific as you can and err on the side of providing too much information.)

* Describe what you'd like the new feature to do.

> (There's no need to design it but suggestions are appreciated.)

**Comment by barnson** [comment]
Windows Installer is in charge of the UI at runtime. WiX is the build tool and cannot influence runtime behavior except as MSI allows. MSI would need to implement a change like this.
